File: base/src/main/java/de/hipp/pnp/base/converter/persistent/CharacterConverter.java
package de.hipp.pnp.base.converter.persistent;

import de.hipp.pnp.api.fivee.abstracts.BaseCharacter;
import jakarta.persistence.Converter;
import org.springframework.stereotype.Component;

@Converter(autoApply = true)
@Component
public class CharacterConverter extends BaseConverter<BaseCharacter> {

    public CharacterConverter() {
        this.type = new BaseCharacter();
    }
}

File: base/src/main/kotlin/de/hipp/pnp/base/converter/persistent/CharacterConverter.kt
package de.hipp.pnp.base.converter.persistent

import de.hipp.pnp.api.fivee.abstracts.BaseCharacter
import jakarta.persistence.Converter
import org.springframework.stereotype.Component

@Converter(autoApply = true)
@Component
class CharacterConverter : BaseConverter<BaseCharacter>() {
    init {
        this.type = BaseCharacter()
    }
}
